### PR TITLE
desmume: Add version 0.9.13

### PR DESCRIPTION
**Pull request by ViCrack**
Closes #[7158 ](https://github.com/ScoopInstaller/Extras/issues/7158)

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

> the sourceforge package is still 0.9.11 out of date

### CODE REVIEW

**Comment by ViCrack** [comment]
https://github.com/ScoopInstaller/Extras/pull/9106

File: bucket/desmume.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "0.9.13",
+    "description": "DeSmuME is a Nintendo DS emulator",
```
Comment (Calinou) [inline bucket/desmume.json:3]:
```suggestion
    "description": "Nintendo DS emulator",
```

File: bucket/desmume.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                ]
+            ]
+        }
+    },
+    "pre_install": "if (!(Test-Path \"$persist_dir\\desmume.ini\")) { New-Item \"$dir\\desmume.ini\" | Out-Null }",
+    "persist": "desmume.ini",
+    "checkver": {
+        "url": "https://api.github.com/repos/TASEmulators/desmume/releases",
+        "regex": "/desmume-([\\d.]+)-win64\\.zip"
+    },
```
Comment (Calinou) [inline bucket/desmume.json:23]: Can't the standard `github` checkver be used here?

Comment (BlackPowerade) [reply bucket/desmume.json:23]: Doesn't look like it. Release tags aren't normal format. `release_0_9_13`

**Pull request closed without merging**